src/nagendra/IdenticalArray.java

```diff
@@ -0,0 +1,29 @@
+package nagendra;
+
+public class IdenticalArray {
+
+	void verifyIdenticalArray(int[] array1, int[] array2) {
+		boolean flag = true;
+		if (array1.length == array2.length) {
+			for (int i = 0; i < array1.length; i++) {
+				if (array1[i] == array2[i])
```

Comment:
You can put condition like below
if (array1[i] != array2[i]){
      flag = false;
      break;
}